setup/tinyHIPPOstart

```diff
@@ -10,13 +10,25 @@
 opkg update
```

Comment:
General comment: Noticed that the base path /etc/tinyHIPPO is used a ton, we should pull this out to an environment variable and reference it in other parts of the code instead of repeating it.